ppcs/ppc0025-perl-version.md

```diff
@@ -0,0 +1,165 @@
+# Perl 5 is Perl
+
+## Preamble
+
+    Author:  Aristotle Pagaltzis <pagaltzis@gmx.de>
+    ID:      ARISTOTLE
+    Status:  Draft
+
+## Abstract
+
+There is no Perl&#160;6 looming any more, so the only thing “Perl” now refers to is Perl&#160;5. That has also forever been the language people mean when they say just “Perl” rather than specifying “Perl&#160;5”. There were earlier versions of Perl, but they are ancient history. All the history that created a distinction between “Perl” and “Perl&#160;5” has fallen away while nobody was paying conscious attention.
```

Comment:
grammar nit:

```suggestion
There is no Perl&#160;6 looming any more, so the only thing “Perl” now refers to is Perl&#160;5. That has also forever been the language that people intend when they say just “Perl” rather than specifying “Perl&#160;5”. There were earlier versions of Perl, but they are ancient history. All the history that created a distinction between “Perl” and “Perl&#160;5” has fallen away while nobody was paying conscious attention.
```

Reply:
I did some googling which confirmed my sense that using “mean” in this way is completely grammatical and standard, and I really don’t like the slightly different meaning of the version with “intend”. So I left out the “intend” bit of this suggestion.

I did however take the “that” bit – the sentence is indeed harder to parse without it. Good catch, thanks.